### DISCUSSION TITLE
[Feature Request] Direct image export?

### DISCUSSION THREAD

**Discussion by Gumball2415** [rafael-fuente/Diffraction-Simulations--Angular-Spectrum-Method#24] [Ideas]
I'm having fun with this simulator thus far, but I often find myself trying to export the direct result instead of the usual image save, which entails having to crop them away by setting the borders to an extreme amount.

Even then, the borders still show at the top and left edges, and it doesn't save the full resolution of the result, which seems to be limited by the resolution of my monitor.
![apeture_diffract_2](https://user-images.githubusercontent.com/54422576/183818041-44a6ced8-e844-46b4-83bc-5da8e21b0b81.png)

**Comment by rafael-fuente** [comment]
This could be easily implemented. 
Note, however, that you can export the images with a higher or lower resolution by using the argument  `figsize` in the visualization methods, for example:

```python
F.plot_colors(rgb, figsize = (15,15))
```

**Comment by LtqxWYEG** [comment]
Here's how: https://github.com/rafael-fuente/diffractsim/issues/50#issuecomment-2408632282
